news/_posts/2019-21-01-resilient-php-applications.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "Resilient PHP applications"
+subtitle: "Introducing the PrestaShop Circuit Breaker library"
+date:   2019-01-21 10:00:00
+authors: [ mandrieu ]
+icon: icon-blaster
+tags:
+ - 1.7
+---
+
+Using an extensible e-commerce platform like PrestaShop is awesome for both developers and e-merchants. Relying on an open source project with the possibility to add themes and plugins available in the store allows every business to create a shop that fits their needs.
+
+But, no matter how well designed is your project and how many tests you have, at some point your website can become unavailable.
+
+This can be because of your database, or calls to third party services (payment methods API, PrestaShop REST API to retrieve modules information) are unavailable.
```

Comment:
```suggestion
This can be because of your database, or calls to third party services are unavailable (e.g: payment methods API, PrestaShop REST API to retrieve modules information, synchronization with an ERP, automatisation of warehouse & logistic, ...)
```